src/Nonlinear/ReverseAD/forward_over_reverse.jl

```diff
@@ -127,7 +85,40 @@ function _eval_hessian_inner(
     return length(ex.hess_I)
 end
 
-function _hessian_slice_inner(d, ex, input_ϵ, output_ϵ, ::Type{T}) where {T}
+function _eval_hessian_chunk(
+    d::NLPEvaluator,
+    ex::_FunctionStorage,
+    offset::Int,
+    chunk::Int,
+    ::Val{CHUNK},
+) where {CHUNK}
+    for r in eachindex(ex.rinfo.local_indices)
+        # set up directional derivatives
+        @inbounds idx = ex.rinfo.local_indices[r]
+        # load up ex.seed_matrix[r,k,k+1,...,k+remaining-1] into input_ϵ
+        for s in 1:chunk
+            # If `chunk < CHUNK`, leaves junk in the unused components
+            d.input_ϵ[(idx-1)*CHUNK+s] = ex.seed_matrix[r, offset+s-1]
```

Comment:
@odow Any reason why we don't use `@inbounds` here ?

Reply:
I would be in favor of removing most `@inbounds`. They made more of a difference in earlier Julia versions.

Reply:
Makes sense, that would be a different PR then anyway